trunk/NDHMS/Routing/module_NWM_io_dict.F

```diff
@@ -2496,18 +2496,18 @@ subroutine initGwDict(gwOutDict)
    !gwOutDict%lonUnits = "degrees_east"
    !gwOutDict%lonStName = "longitude"
 
-   gwOutDict%varNames(:) = [character(len=64) :: 'inflow','outflow','depth']
-   gwOutDict%longName(:) = [character(len=64) :: 'Bucket Inflow','Bucket Outflow','Bucket Depth']
-   gwOutDict%units(:) = [character(len=64) :: 'm3 s-1','m3 s-1','mm']
-   !gwOutDict%coordNames(:) = [character(len=64) :: 'latitude longitude','latitude longitude','latitude longitude']
-   gwOutDict%scaleFactor(:) = [0.01,0.01,0.1]
-   gwOutDict%addOffset(:) = [0.0,0.0,0.0]
-   gwOutDict%outFlag(:) = [0,0,0]
-   gwOutDict%timeZeroFlag(:) = [0,0,1]
-   gwOutDict%fillReal(:) = [-9999.0,-9999.0,-9999.0]
-   gwOutDict%missingReal(:) = [-9999.0,-9999.0,-9999.0]
-   gwOutDict%validMinDbl(:) = [-10000.0d0, -10000.0d0, -10000.0d0]
-   gwOutDict%validMaxDbl(:) = [10000.0d0, 10000.0d0, 10000.0d0]
+   gwOutDict%varNames(:) = [character(len=64) :: 'inflow','outflow','loss','depth']
+   gwOutDict%longName(:) = [character(len=64) :: 'Bucket Inflow','Bucket Outflow','Bucket Loss','Bucket Depth']
+   gwOutDict%units(:) = [character(len=64) :: 'm3 s-1','m3 s-1','m3 s-1','mm']
+   !gwOutDict%coordNames(:) = [character(len=64) :: 'latitude longitude','latitude longitude','latitude longitude','latitude longitude']
+   gwOutDict%scaleFactor(:) = [0.00001,0.00001,0.00001,0.1]
```

Comment:
Is this high, or necessary to avoid step functions?

Reply:
I think something between what it was and these (picked because they are consistent w/ chrt vars) will work.  Thought I had taken these out with the metadata changes and was planning to adjust anyway.